docs/tutorials/otio-serialized-schema.md

```diff
@@ -29,24 +29,24 @@ changes.  If it needs to be updated and this file regenerated, run:
 ```
 Adapters convert between OTIO and other formats.
 
-    Note that this class is not subclassed by adapters. Rather, an adapter is
-    a python module that implements at least one of the following functions:
+Note that this class is not subclassed by adapters. Rather, an adapter is
+a python module that implements at least one of the following functions:
 
-    .. code-block:: python
+.. code-block:: python
 
-        write_to_string(input_otio)
-        write_to_file(input_otio, filepath) (optionally inferred)
-        read_from_string(input_str)
-        read_from_file(filepath) (optionally inferred)
+    write_to_string(input_otio)
+    write_to_file(input_otio, filepath) (optionally inferred)
+    read_from_string(input_str)
+    read_from_file(filepath) (optionally inferred)
```

Comment:
Was this file meant to go into this change?  I'm surprised that adding V2d serialization would result in a change to the serialized schema markdown file.

Reply:
I was following README.md at the root which asked me to run `make test`.  That failed with the message that I needed to run `make doc-model-update`, so I did which changed this whitespace.

I thought I would leave it in so future contributors don't run into the same issue, but I can revert if you prefer.

Reply:
Huh, thats interesting.  let me do a fresh clone and see if something has changed that would cause that... thanks for flagging it.